test/__tests__/ts-decorator-auto-accessor-test.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
Can these go in the existing `src/__tests__` directory?

Reply:
Okay. I moved the test.